lib/master.js

```diff
@@ -23,7 +23,9 @@ function liveWorkerIds() {
       worker.process.exitCode == null;
     return isAlive;
   });
-  return workerIds;
+  return workerIds.sort(function(a, b) {
```

Comment:
It turns out that while js sorts numbers as if they were strings, lodash sorts strings as if they were lexical... and cluster IDs are strings, so lodash sorts as `["10", "9"]`... I should have investigated more thoroughly. PR to fix this is coming.